test/end-to-end/fiscalYears/fiscalYears.spec.js

```diff
@@ -159,4 +159,41 @@ describe('Fiscal Year', function () {
 
   });
 
+  it('closing a fiscal year in normal way', () => {
+    helpers.navigate(path);
+
+    // the last in the list is the oldest
+    let updateButton = element.all(by.css('[data-fiscal-entry]'));
+    updateButton.all(by.css('[data-method="update"]')).last().click();
+
+    // this fix multiple element found take first
```

Comment:
Thanks for the comment.  Perhaps it would be best to increase the specificity of the selector in coming tests.